modules/oe_theme_helper/src/TwigExtension/Filters.php

```diff
@@ -59,24 +59,52 @@ public function toNativeLanguageName($language_code): string {
   }
 
   /**
-   * Get file icon given its extension.
+   * Get file icon class given its extension.
    *
    * @param mixed $extension
    *   File extension.
    *
    * @return string
-   *   File icon.
+   *   File icon class name.
    */
-  public function toFileIcon($extension): string {
-    // @todo: Complete and/or centralise extension to icon conversion.
-    // Get icons from given file extension.
-    switch ($extension) {
-      case 'ppt':
-        return 'presentation';
+  public function toFileIcon($extension) {
```

Comment:
The `$extension` might come in uppercase (sometimes images have `.JPG` instead of `.jpg`). To improve this I would lowercase the `$extension` here so we are sure to find the proper class.

Reply:
Nice remarks !
I would then also suggest to take in account .jpeg and .JPEG

Reply:
Oops, forget what I just said!